react/features/base/media/components/web/Video.js

```diff
@@ -200,4 +206,22 @@ class Video extends Component<Props> {
     }
 }
 
-export default Video;
+/**
+ * Maps (parts of) the Redux state to the associated Video props.
+ *
+ * @param {Object} state - The Redux state.
+ * @private
+ * @returns {{
+ *     noAutoPlayVideo: boolean
+ * }}
+ */
+function _mapStateToProps(state) {
+    const testingConfig = state['features/base/config'];
+
+    return {
+        noAutoPlayVideo: testingConfig && testingConfig.noAutoPlayVideo
```

Comment:
Isn't this going to be actually `config.noAutoPlayVideo` and not `config.testing.noAutoPlayVideo`?

Reply:
You're absolutely right, I forgot to updated my deployment for testing after I modified the code with your previous recommendation.